HeterogeneousCore/CUDATest/test/BuildFile.xml

```diff
@@ -5,5 +5,10 @@
     <use name="catch2"/>
   </bin>
 
-  <test name="TestCUDATest" command="runtests.sh"/>
+  <ifrelease name="_GPU_">
+    <test name="TestCUDATest" command="runtests.sh gpu"/>
+  <else/>
+    <test name="TestCUDATest" command="runtests.sh cpu"/>
+  </ifrelease>
```

Comment:
One of the tests sets the accelerator to GPU, so the job should succeed with a GPU and fail without. The test is set up to test both outcomes. My attempt here is in the `GPU_X` IBs run the test requiring a GPU, and elsewhere base that decision whether a GPU is available or not (via `cudaIsEnabled` program). My first thought was to expect the job to always fail elsewhere, but then the test would fail on machines that have a GPU.

@smuzaffar Is this the best way to achieve that?

Reply:
yes looks right